app/components/mandatarissen/update-state.hbs

```diff
@@ -57,6 +57,46 @@
       >De gekozen datum moet na de startdatum liggen van het huidige mandaat.</AuAlert>
     {{/unless}}
   </div>
+  {{#unless this.isTerminatingMandate}}
+    <div>
+      <AuLabel>
+        Beleidsdomeinen
+      </AuLabel>
+      <Mandatarissen::BeleidsdomeinSelectorWithCreate
+        @beleidsdomeinen={{this.selectedBeleidsdomeinen}}
+        @onSelect={{this.updateBeleidsdomeinen}}
+      />
+      <AuHelpText>Enkel voor burgemeester en schepenen</AuHelpText>
+    </div>
+    <div>
+      <AuLabel>
+        Fractie
+      </AuLabel>
+      {{#if this.loading}}
+        Loading...
```

Comment:
Bit weird that this has a separate loading message, since the beleidsdomeinselectorwithcreate does not and since this component contains a loading state.

Reply:
the reason is that this is the only thing that really can't be used while the component is loading, all other fields can be filled out